src/DotNetWorker.Core/DotNetWorker.Core.csproj

```diff
@@ -26,12 +26,12 @@
         <PackageReference Include="Azure.Core" Version="1.10.0" />
         <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
         <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="5.0.0" />
-        <PackageReference Include="System.Collections.Immutable" Version="6.0.0" />
+        <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
     </ItemGroup>
 
     <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-        <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0" />
+        <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="5.0.0" />
         <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
-        <PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
+        <PackageReference Include="Microsoft.Extensions.Options" Version="5.0.0" />
```

Comment:
Just curious why the downgrade in version?

Reply:
These packages were added in previous PR as part of adding .NET standard support. But while testing changes for this PR, I ran into assembly version conflict issue where the CLR loads the 5.X version of an assembly, and the runtime expects the 6.X version of the assembly.  I synced with @fabiocav to discuss this issue and we decided to downgrade these packages to 5X version.

Here is one specific exception message I ran into:
> "Could not load file or assembly 'Microsoft.Extensions.DependencyInjection.Abstractions, Version=6.0.0.0

and I could see that 5.X version is loaded.